### PR TITLE
Update .htaccess

### PR DESCRIPTION
**Pull request by SebSeis**
Added modules and extension of the ontology.

### CODE REVIEW

**Comment by dgarijo** [comment]
Please have a look at my comment.

File: ocqa/.htaccess [hunk 1]
```diff
@@ -25,3 +25,27 @@ RewriteRule ^ocqa.html$ https://sebseis.github.io/OCQA/ [R=302,NE,L]
...
+RewriteRule ^regulation.ttl$ https://sebseis.github.io/OCQA/Modules/Regulation/OCQA-Regulation.ttl [R=302,NE,L]
+RewriteRule ^regulation.html$ https://sebseis.github.io/OCQA/Modules/Regulation/ [R=302,NE,L]
+# Rule Module
+RewriteRule ^rule.ttl$ https://sebseis.github.io/OCQA/Modules/Rule/OCQA-Rule.ttl [R=302,NE,L]
+RewriteRule ^rule.html$ https://sebseis.github.io/OCQA/Modules/Rule/ [R=302,NE,L]
+
+# Link to Extensions, already described in main document
+# Screed Extension
+RewriteRule ^screed.ttl$ https://sebseis.github.io/OCQA/Extensions/Screed/OCQA-Screed.ttl [R=302,NE,L]
+RewriteRule ^rule.html$ https://sebseis.github.io/OCQA/Extensions/Screed/ [R=302,NE,L]
```
[the remark this answers is not in the document]
Comment (davidlehn) [reply ocqa/.htaccess:46]: Looks like a typo.
```suggestion
RewriteRule ^screed.html$ https://sebseis.github.io/OCQA/Extensions/Screed/ [R=302,NE,L]
```

File: ocqa/.htaccess [hunk 2]
```diff
@@ -25,3 +25,27 @@ RewriteRule ^ocqa.html$ https://sebseis.github.io/OCQA/ [R=302,NE,L]
...
+# ThermalInsulation Extension
+RewriteRule ^thermalInsulation.ttl$ https://sebseis.github.io/OCQA/Extensions/Insulation/OCQA-ThermalInsulation.ttl [R=302,NE,L]
+RewriteRule ^rule.html$ https://sebseis.github.io/OCQA/Extensions/Insulation/ [R=302,NE,L]
```
Comment (davidlehn) [inline ocqa/.htaccess:49]:
```suggestion
RewriteRule ^thermalInsulation.html$ https://sebseis.github.io/OCQA/Extensions/Insulation/ [R=302,NE,L]
```

Comment (SebSeis) [reply ocqa/.htaccess:49]: Thank you for being so helpful. You were right; it is a typo. I am not familiar with the htaccess files and used language, and I hope that the changes will work as I wanted. In our case, we used some modules and extensions, which we try to provide for users.

**Review by SebSeis — COMMENTED** [review]
Thanks for correcting the typo.

**Pull request merged**